packages/loader/container-loader/src/test/deltaManager.spec.ts

```diff
@@ -104,6 +105,23 @@ describe("Loader", () => {
                 } as any as ISequencedDocumentMessage;
             }
 
+            async function emit(count: number, type: MessageType) {
```

Comment:
given that function mimics (I think) sending ops and receiving acks for them, I'd rename it to reflect it. Maybe sendAndReceiveOps(), or something like that